plugins/webkul/employees/src/Filament/Clusters/Configurations/Resources/ActivityPlanResource.php

```diff
@@ -55,7 +56,20 @@ public static function form(Form $form): Form
                             ->searchable()
                             ->preload()
                             ->createOptionForm(fn (Form $form) => CompanyResource::form($form))
-                            ->editOptionForm(fn (Form $form) => CompanyResource::form($form)),
+                            ->editOptionForm(fn (Form $form) => CompanyResource::form($form))
+                            ->afterStateHydrated(function (Forms\Components\Select $component, $state) {
```

Comment:
We should show the option with (Deleted) suffix and these options should disabled as well